src/styles/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  width: 200px;
+  padding: 16px;
```

Comment:
Padding top is 32px but not 16

![image](https://user-images.githubusercontent.com/63306833/173331155-1fd4c484-5deb-4272-929b-3685aba723c0.png)